### PR TITLE
 [HotFix] removing user data after logout

### PR DESCRIPTION
**Pull request by marckraw**

### CODE REVIEW

File: src/user/reducers/index.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 export function user(state = initialState, action) {
     switch (action.type) {
         case actions.GET_USER_DATA_FULFILLED:
-            console.log(action.payload);
             return {
                 ...state,
                 ...action.payload.data,
             };
+        case authActions.LOGOUT_USER_FULFILLED:
+            return {
```
Comment (tomaszferens) [inline src/user/reducers/index.js]: `return initialState` is enough - no need to make another object :)

Comment (marckraw) [reply src/user/reducers/index.js]: 👍

File: src/components/NonAuthenticatedRoutes/index.jsx [hunk 2]
```diff
@@ -10,28 +10,27 @@ import AfterRegister from 'pages/Register/AfterRegister';
...
+        render={() =>
+            rest.authenticated ? (
+                <Redirect to="/main" />
+            ) : (
+                <Fragment>
+                    <Switch>
+                        <Route exact path="/login" component={Login} />
+                        <Route path="/about" component={About} />
+                        <Route path="/register" component={Register} />
+                        <Route path="/afterRegister" component={AfterRegister} />
```
Comment (tomaszferens) [inline src/components/NonAuthenticatedRoutes/index.jsx]: I don't think we should name routes in `camelCase`. According to SO, it should be `hyphenated`, so `after-register`. Also that helps Google bot a bit ;)

**Review by tomaszferens — APPROVED** [review]

File: src/components/NonAuthenticatedRoutes/index.jsx (hunk 2 above)
Comment (marckraw) [reply src/components/NonAuthenticatedRoutes/index.jsx]: Sure, didn't know about that ;) i will change then in every place to hyphenated ones.

**Pull request merged**